modules/dynamic-provisioning-azure-disk-definition.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * storage/dynamic-provisioning.adoc
+
+[id="azure-disk-definition-{context}"]
+= Azure Disk object definition
+
+.azure-advanced-disk-storageclass.yaml
+[source,yaml]
+----
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: slow
+provisioner: kubernetes.io/azure-disk
+parameters:
+  storageAccount: azure_storage_account_name  <1>
+  storageaccounttype: Standard_LRS  <2>
+  kind: Dedicated  <3>
+----
+<1> Azure storage account name. This must reside in the same resource 
+group as the cluster. If a storage account is specified, the `location` 
+is ignored. If a storage account is not specified, a new storage 
+account gets created in the same resource group as the cluster. If you 
+are specifying a `storageAccount`, the value for `kind` must be `Dedicated`.
+<2> Azure storage account SKU tier. Default is empty. *Note:* Premium 
```

Comment:
I don't think that we normally do inline NOTE's like this. I would just say "Note that premium VMs..."

Reply:
I've adjusted this to use the proposed text.